### DISCUSSION TITLE
visread docs link is dead

### DISCUSSION THREAD

**Discussion by trq5014** [MPoL-dev/MPoL#86] [Q&A]
The link to the visread docs from [https://mpol-dev.github.io/](https://mpol-dev.github.io/) is not working. It sends you to [https://visread.readthedocs.io/](https://visread.readthedocs.io/) and from my knowledge we no longer use read the docs. This is minor, but I figured I would bring it to attention.

**Comment by iancze** [comment]
Ah yea, that's a good catch. The new link is https://mpol-dev.github.io/visread/
